ff/src/fields/models/fp/montgomery_backend.rs

```diff
@@ -435,60 +430,61 @@ impl<T: MontConfig<N>, const N: usize> FpConfig<N> for MontBackend<T, N> {
     }
 }
 
-impl<T, const N: usize> Fp<MontBackend<T, N>, N> {
+impl<T: MontConfig<N>, const N: usize> Fp<MontBackend<T, N>, N> {
+    /// Construct a new field element from its underlying
+    /// [`struct@BigInt`] data type.
+    #[inline]
+    const fn new(element: BigInt<N>) -> Self {
+        let mut r = Self(element, PhantomData);
+        if r.const_is_zero() {
+            r
+        } else {
+            r = r.mul(&Fp(T::R2, PhantomData));
+            r
+        }
+    }
+
+    /// Construct a new field element from its underlying
+    /// [`struct@BigInt`] data type.
+    ///
+    /// Unlike [`Self::new`], this method does not perform Montgomery reduction.
```

Comment:
And also: since we're currently referring to a private method within the docs of a public method, this produces a cargo warning. I'd avoid these as it might increase in severity in future releases.